### PR TITLE
Styling_imports-fix

### PR DESCRIPTION
**Pull request by kotesh-arya**
- Updated main, section elements with dedicated classes.

- Refactored documentation styling into docs.css.

- Refactored components.css file with imported individual component style-sheets.

### CODE REVIEW

File: Components/Alert/alert.html [hunk 1]
```diff
@@ -106,8 +108,8 @@ <h3>Components</h3>
                 </div>
             </article>
             <div class="section-footer">
-                <a href="/Components/Avatar/avatar.html"> < Avatar</a>
-                <a href=""> Badge > </a>
+                <a class="section-footer-link" href="/Components/Avatar/avatar.html"> < Avatar</a>
+                <a class="section-footer-link" href=""> Badge > </a>
```
Comment (TejasShekar) [inline Components/Alert/alert.html]: Add working links in all your components instead of empty href.

**Review by TejasShekar — APPROVED** [review]
Go through your CL once and add page links in your footer navigation.
Rest looks good to merge.

File: Components/Alert/alert.html (hunk 1 above)
Comment (kotesh-arya) [reply Components/Alert/alert.html]: Sure, WIll look in and update them 👍🏻

File: Components/Avatar/avatar.html [hunk 2]
```diff
@@ -7,11 +7,12 @@
...
     <meta name="author" content="Kotesh Mudila">
     <meta name="description" content="Handy-UI documentation - Avatar component">
-    <link rel="stylesheet" href="avatar.css">
-    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/index.css">
     <link rel="stylesheet" href="/css/header.css">
     <link rel="stylesheet" href="/css/sidebar.css">
     <link rel="stylesheet" href="/css/components.css">
+    <link rel="stylesheet" href="/css/docs.css">
+
```
Comment (sks-21) [inline Components/Avatar/avatar.html:15]: You can make one combined CSS file and import all these files there. And just use one link tag for importing that combined CSS file

File: Components/Badge/badge.html [hunk 3]
```diff
@@ -7,11 +7,12 @@
...
     <meta name="author" content="Kotesh Mudila">
     <meta name="description" content="Handy-UI documentation - Badge component">
-    <link rel="stylesheet" href="/Components/Badge/badge.css">
-    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/index.css">
     <link rel="stylesheet" href="/css/header.css">
     <link rel="stylesheet" href="/css/sidebar.css">
     <link rel="stylesheet" href="/css/components.css">
+    <link rel="stylesheet" href="/css/docs.css">
+
```
Comment (sks-21) [inline Components/Badge/badge.html:15]: same as above

File: Components/Button/button.html [hunk 4]
```diff
@@ -7,11 +7,12 @@
...
     <meta name="author" content="Kotesh Mudila">
     <meta name="description" content="Handy-UI documentation - Button component">
-    <link rel="stylesheet" href="/Components/Button/button.css">
-    <link rel="stylesheet" href="/index.css">
+    <link rel="stylesheet" href="/css/index.css">
     <link rel="stylesheet" href="/css/header.css">
     <link rel="stylesheet" href="/css/sidebar.css">
     <link rel="stylesheet" href="/css/components.css">
+    <link rel="stylesheet" href="/css/docs.css">
+
```
Comment (sks-21) [inline Components/Button/button.html:15]: same as above

File: Components/Button/button.html [hunk 5]
```diff
@@ -57,7 +58,7 @@ <h3>Components</h3>
                 <li><a class="sidebar-item" href="/Components/Navigation/navigation.html">Navigation</a></li>
             </ul>
         </div>
-        <section class="button-section">
+        <section class="section-content">
```
Comment (sks-21) [inline Components/Button/button.html:61]: use prettier to format code

**Review by sks-21 — APPROVED** [review]
Just go through the suggestions once. Rest everything looks good.

File: Components/Badge/badge.html (hunk 3 above)
Comment (kotesh-arya) [reply Components/Badge/badge.html:15]: Sure

File: Components/Avatar/avatar.html (hunk 2 above)
Comment (kotesh-arya) [reply Components/Avatar/avatar.html:15]: Sure , Will follow that 👍🏻

**Pull request merged**